internal/pkg/cli/env_init.go

```diff
@@ -767,6 +782,28 @@ func (o *initEnvOpts) validateCredentials() error {
 	return nil
 }
 
+func (o *initEnvOpts) validateInternalALBSubnets() error {
+	if len(o.internalALBSubnets) == 0 {
+		return nil
+	}
+	isImported := make(map[string]bool)
+	for _, placementSubnet := range o.internalALBSubnets {
```

Comment:
nit: this is O(N^2). This can be O(N) if
1. all internalALBSubnets to a map
2. Go over subnets. If a given subnet is present in the map above, then add to isImported

Reply:
Thinking about this more, this can be simpler than above. The new map can be called `missing` and in the loop (2) remove items from the map. Then in the L797 check if the map isn't empty, i.e. something is still missing